testing/btest/Baseline/core.record-field-defer-table-2/output

```diff
@@ -0,0 +1,19 @@
+### BTest baseline data generated by btest-diff. Do not edit. Use "btest -U/-u" to update. Requires BTest >= 0.63.
+0, my_seq()
+1, my_seq()
+2, my_seq()
+3, my_seq()
+4, printing
+4, s1, [seq=1, inner=[seq=3]]
+4, s2, [seq=2, inner=[seq=4]]
+4, tbl, {
+4, my_seq()
```

Comment:
I haven't looked what's happening here, but printing an element invokes the `&default` expression 3 times :man_shrugging: 

This happens with vanilla 6.0 and 5.2.

Reply:
Explanation/fix in #3270.